code/game/machinery/from_beyond.dm

```diff
@@ -0,0 +1,99 @@
+/obj/machinery/from_beyond
+	name = "resonance wave emissor"
+	desc = "We see things only as we are constructed to see them."
+	icon = 'icons/obj/xenoarchaeology.dmi'
+	icon_state = "ano60"	//TODO: get unique sprites for this
+
+	anchored = 0
+	density = 1
+
+	use_power = 1
+
+	active_power_usage = 1 KILOWATTS
+	active_power_usage = 10 KILOWATTS
+
+	var/active = FALSE
+	var/list/whispers = list(
+			"You hear something behind you.",
+			"Endless eyes gazes upon you from the above.",
+			"All is gone.",
+			"The light is slowly dying.",
+			"This world is not yours, who are you?!",
+			"Blasphemous whispers invades your thoughts.",
+			"You hear a nasty ripping noise, as if flesh is being torn apart.",
+			"All hope is lost.",
+			"The abyss stares back at you.",
+			"The rats, the rats in the walls!",
+			"The air is filled with poisonous whispers and cruel truths.",
+			"The stars are gone, all is void now.",
+			"He comes...",
+			"The end is near.",
+			"You are all alone.",
+			"You feel like a stranger in a strange land.",
+			"The world crumbles under your feet.",
+			"You see worlds that no other living men have seen.",
+			"There is no escape..."
+			)
+
+/obj/machinery/from_beyond/attack_hand(var/mob/living/carbon/human/user as mob)
+	usr.set_machine(src)
+
+	if(!active)
+		src.visible_message("<span class='warning'>[user] switches \the [src] on.</span>")
+		usr << "<span class='warning'>The world beyond opens to your eyes.</span>"
+		active = TRUE
+
+	else
+		src.visible_message("<span class='warning'>[user] switches \the [src] off.</span>")
+		usr << "<span class='warning'>The world beyond vanishes before your eyes.</span>"
+		active = FALSE
+
+	update_icon()
+
+/obj/machinery/from_beyond/update_icon()
+	if(!(stat & NOPOWER))
+		icon_state = "ano60"
+
+	if(active)
+		icon_state = "ano61"
+
+	else
+		icon_state = "ano60"
+
+	return
+
+/obj/machinery/from_beyond/machinery_process()
+	..()
+	if(active)
+
+		for(var/obj/machinery/light/P in view(7, src))
+			P.flicker(1)
```

Comment:
Every 2 seconds, every light within 7 tiles flickers? Maybe add a prob to this to make it a bit less frequent.